docs/blog/2018-1-18-strapi-and-gatsby/index.md

```diff
@@ -1,192 +1,272 @@
 ---
 title: "Building a static blog using Gatsby and Strapi"
-date: 2018-01-18
+date: 2018-1-18
 author: "Pierre Burgy"
-tags: ["strapi"]
+canonicalLink: https://blog.strapi.io/building-a-static-website-using-gatsby-and-strapi/
+tags: ["strapi", "blog", "cms", "headless-cms", "open source", "tutorials"]
 ---
 
-## Introduction
+(Article updated on 20 March 2019. This updates the tutorial code examples. We added videos with additional explanations to each section.)
 
-A static website contains Web pages with fixed content. Technically, it is a simple list of HTML files, which displays the same information to every visitor. Unlike dynamic websites, they do not require any backend programming or database. Publishing a static website is easy: the files are uploaded on a simple Web server or storage provider. The two main advantages of static websites are security and speed: there is no database so it can not be hacked and there is no need to render a page for each request, which makes Web browsing faster.
+![Gatsby Strapi](gatsby-strapi.png)
```

Comment:
More descriptive alt text would be cool for screenreader users (applies to all images)